dom.bs

```diff
@@ -3594,6 +3623,60 @@ run these steps:
 <ol>
  <li><p>Assert: either <var>addedNodes</var> or <var>removedNodes</var> <a for=set>is not empty</a>.
 
+ <li><p>If either <var>options</var>'s' {{MutationObserverInit/elementLocalNameFilter}} or
+ {{MutationObserverInit/elementHasAttributeFilter}} is present, then:
+
+ <ol>
+  <li><p>Let <var>nodeAttributesList</var> be <var>nodeAttributesList</var> of the node's attributes, "<code>node.attributes.keys</code>".
```

Comment:
`node.attribute.keys` may not be accurate. Try to find the appropriate API to capture the attributeNames of a node.

Reply:
Possibly `getAttributeNames`: https://developer.mozilla.org/en-US/docs/Web/API/Element/getAttributeNames

Seems like `element.getAttributeNames()` is rather new-ish. Not sure if there is a preference on choosing APIs or code that has more natural backwards compatibility. 

Should I use the handy API `getAttributeNames`, or in the code block, write the polyfill code for it?